ivy/functional/frontends/paddle/nn/functional/norm.py

```diff
@@ -6,13 +6,23 @@
 
 @to_ivy_arrays_and_back
 @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
-def layer_norm(x, normalized_shape, weight=None, bias=None, epsilon=1e-05, name=None):
-    return ivy.layer_norm(x, normalized_shape, weight, bias, epsilon)
+def batch_norm(
+    x,
+    mean,
+    variance,
+    offset=None,
+    scale=None,
+    training=False,
+    epsilon=1e-5,
+    momentum=1e-1,
+    data_format="NSC",
+):
+    return ivy.batch_norm(
+        x, mean, variance, offset, scale, training, epsilon, momentum, data_format
+    )
 
 
 @to_ivy_arrays_and_back
 @with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
-def normalize(x, p=2, axis=1, epsilon=1e-12, name=None):
-    if axis < 0:
-        axis = ivy.get_num_dims(x) + axis
-    return ivy.lp_normalize(x, p=p, axis=axis)
+def layer_norm(x, normalized_shape, weight=None, bias=None, epsilon=1e-05, name=None):
+    return ivy.layer_norm(x, normalized_shape, weight, bias, epsilon)
```

Comment:
Could you undo all the changes you have made to `layer_norm` and `normalize` function and only add `batch_norm` function.

Reply:
Sorry about that. I see that layer_norm didn't get any change, and still here. I added normalization back. I didnt touch theses functions I justed on batch_norm, but I am sure why they got changed. Please let me know about layer_norm becuase I see it here. Thank you.